db/migrations/DB.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE "peeps" (
```

Comment:
This does not set up the TABLE as you expect it to. The "" are not needed, and capitalisation needs checking: 

CREATE TABLE peeps(id SERIAL PRIMARY KEY ...